os/pm/pm_timedstay.c

```diff
@@ -78,11 +78,11 @@ static WDOG_ID pid_timer_map[CONFIG_MAX_TASKS];
  * Private Functions
  ************************************************************************/
 
-static void timer_timeout(int argc, uint32_t state, uint32_t pid)
+static void timer_timeout(int argc, uint32_t domain, uint32_t pid)
 {
 	/* PM transition will be relaxed here */
 	if (pid_timer_map[pid] != NULL) {
-		pm_relax(PM_IDLE_DOMAIN, (enum pm_state_e)state);
+		pm_relax(domain);
 		wd_delete(pid_timer_map[pid]);
```

Comment:
Please chenge this map using pid to using domain.

Reply:
@gSahitya-samsung i think this is still not updated . Can you look into it?